hikari/internal/cache.py

```diff
@@ -70,7 +70,7 @@
 """Type-hint for mapping values."""
 
 
-class StatefulCacheMappingView(cache.CacheView[KeyT, ValueT], typing.Generic[KeyT, ValueT]):
+class CacheMappingView(cache.CacheView[KeyT, ValueT], typing.Generic[KeyT, ValueT]):
```

Comment:
did we both coincidentally make the exact same name change in two separate draft prs? 👀

Reply:
lol